Competition2017/tests/competition/subsystems/drive/DriveTestBase.java

```diff
@@ -30,4 +31,14 @@ public void verifyDriveSetpoints(double left, double right) {
         assertEquals(left, ((MockCANTalon)drive.leftDrive).getSetpoint(), 0.001);
         assertEquals(right, ((MockCANTalon)drive.rightDrive).getSetpoint(), 0.001);
     }
+    
+    public void verifyDrivePositive() {
+        assertTrue(((MockCANTalon)drive.leftDrive).getSetpoint() > 0);
```

Comment:
Is there a comparison assertion method? I can't remember if there is with JUnit.